README.md

```diff
@@ -97,12 +117,21 @@ If you're looking for how to download, build, install and run Helios, keep readi
 Prerequisities
 --------------
 
+The binary release of Helios is built for Ubuntu 14.04.1 LTS, but Helios should
```

Comment:
Well, deb packaging is not a proper standard, so the fact that you sometimes can install deb packages on a distribution it was not built for is more of a happy coincidence than something you can expect. Because of this I think it's a good idea to be explicit about what target environment we're building for.